project_euler/euler_prob_2/euler2.rb

```diff
@@ -0,0 +1,25 @@
+# Description:  add previous two numbers to get next number in  sequence, find all even numbers under 4 million, add them
+# https://projecteuler.net/problem=2
+
+
```

Comment:
This looks good! I like how you filtered out the even numbers and added everything together.  I'd like to revisit this and try to re-factor without saving anything in an array.  Could you use something else to keep track of the sum of the even numbers as you go through the sequence?